test_problems/fortran/f77_ctlib_blessed.txt

```diff
@@ -1,53 +1,53 @@
-           12   1.8867924809455872E-002   -1682714.294998080     
-           13   1.8867924809455872E-002    37464941.77507096     
-           14   1.8867924809455872E-002   -3392174.599151304     
-           15   1.8867924809455872E-002    18021024.20126612     
-           16   1.8867924809455872E-002    463920.1244349164     
-           17   1.8867924809455872E-002    2806717.017385867     
-           18   1.8867924809455872E-002   -1667616.834202130     
-           19   1.8867924809455872E-002    1880348.120627985     
-           20   1.8867924809455872E-002   -8475594.985827351     
-           21   1.8867924809455872E-002   -6400868.095467533     
-           22   1.8867924809455872E-002   -1406782.679495723     
-           23   1.8867924809455872E-002    13553530.31813303     
-           24   1.8867924809455872E-002    773602.1497574156     
-           25   1.8867924809455872E-002   -1655899.720775956     
-           26   1.8867924809455872E-002   -1169234.881460521     
-           27   1.8867924809455872E-002   -5423450.042600359     
-           28   1.8867924809455872E-002   -4477253.293424521     
-           29   1.8867924809455872E-002    1678803.902351405     
-           30   1.8867924809455872E-002   -7621437.918464270     
-           31   1.8867924809455872E-002    2920878.667345533     
-           32   1.8867924809455872E-002    1587099.067790282     
-           33   1.8867924809455872E-002   -1115914.875271953     
-           34   1.8867924809455872E-002   -1114194.976870384     
-           35   1.8867924809455872E-002   -25381975.04578404     
-           36   1.8867924809455872E-002    19123070.07861118     
-           37   1.8867924809455872E-002   -5098323.231374484     
-           38   1.8867924809455872E-002   -437061.2305954426     
-           39   1.8867924809455872E-002   -14456790.76068574     
-           40   1.8867924809455872E-002   -2054215.955030773     
-           41   1.8867924809455872E-002    5671696.880337518     
-           42   1.8867924809455872E-002   -877769.9776598437     
-           43   1.8867924809455872E-002   -5388299.031743989     
-           44   1.8867924809455872E-002   -314212.4087394696     
-           45   1.8867924809455872E-002   -6148936.211230934     
-           46   1.8867924809455872E-002    3674698.329455073     
-           47   1.8867924809455872E-002   -2633122.304698291     
-           48   1.8867924809455872E-002    31625354.14713462     
-           49   1.8867924809455872E-002  -2.3592239273284576E-013
-           50   1.8867924809455872E-002    2079545.452265354     
-           51   1.8867924809455872E-002   -6636261.786209078     
-           52   1.8867924809455872E-002   -2239463.603775544     
-           53   1.8867924809455872E-002   -528516.5887338896     
+           1   1.8867924809455872E-002   107579642.89353865     
+           2   1.8867924809455872E-002  -127616418.89938366     
```

Comment:
All these changes seem like they should be below the test tolerances. Was there something that was actually causing a failure?

Reply:
There was a change in `AR`, which previously had a spurious number but now is exactly `0`, causing a failure. I.e. I replaced the blessed output.